apisix/utils/log-util.lua

```diff
@@ -56,6 +56,12 @@ local function get_full_log(ngx)
         start_time = ngx.req.start_time() * 1000,
         latency = (ngx.now() - ngx.req.start_time()) * 1000
     }
+
+    if conf.include_req_body then
+        log.request.body = ngx.req.get_body_data()
```

Comment:
Should we distinguish `without body` (like `GET` request) from `body too big` (like `get_body_data()` returns `nil` but `get_body_file()` doesn't)?

Reply:
yes, I think we shoud do.

Reply:
agreed, I'll make the changes accordingly.